src/styles/_header.scss

```
@@ -0,0 +1,66 @@
+@import "var";
+
+.header {
+  height: 765px;
+
+  background-image: url(../images/header.jpg);
+  background-size: cover;
+  background-position: center;
```

Запиши це шорткатом в один рядок
```suggestion
  background: url() cover center;
```

---

шорткат не рабоатет